### ISSUE TITLE
Multiobject warning

### ISSUE THREAD

**Issue by PipBrewer** [NHMDenmark/Mass-Digitizer#274]
Christian Lange (22/02/2023) suggested that when doing, multispecimen objects, that have a pop up that asks you to confirm that it is a multispecimen object when try to save record. That behaviour will continue until uncheck multiobject checkbox. The reason for this request, is he (and there was agreement from others) that it would be very easy to overlook the fact that you had forgotten to uncheck the box. This would be harder and more time consuming to fix later. There are also not enough multiobject sheets that the warning would be too troublesome. 

HOWEVER, you are more likely to encounter MANY multiobject specimens in entomology and this pop up box would be very annoying?

For discussion with development team - ideas.

**Comment by FedorSteeman** [comment]
Instead of a popup, it could also be a label (with red letters perhaps) in close proximity to the save button. In any case seems easy to implement. Let's discuss.

**Comment by jlegind** [comment]
I like Fedor's suggestion that there could be a loud and obnoxious label or popup stating that --you are about to save a multi-object record--

**Comment by FedorSteeman** [comment]
Or perhaps a popup Clippy wagging its little end: 
![image](https://user-images.githubusercontent.com/10909008/222443141-634a7bed-0bb8-4d1e-b467-1bccdd39b7f6.png)

**Comment by PipBrewer** [comment]
Bhupjit has alternative suggestions which aren't as annoying than clicking on pop ups

**Comment by jlegind** [comment]
Effort level: 'Easy'

**Comment by jlegind** [comment]
Effort level: 'Easy'

**Comment by jlegind** [comment]
We have some additional ideas that are not pop-up windows/prompts.
It could be:
- Having an active radio button other than SSO, we will highlight the multispecimen section with a heavy contrast color.
- Highlighting the Container ID field itself
- Putting a large warning sticker in the free area on the App UI itself with an appropriate text:

**Comment by jlegind** [comment]
Had a meeting with Chelsea and we came to the conclusion that a sticker or warning sign at the bottom right corner of the app would be very striking. The optimal solution would to have the warning label AND highlighting the Container ID field.

**Comment by jlegind** [comment]
I created this warning label 
![simple_warning_small](https://github.com/NHMDenmark/Mass-Digitizer/assets/5330066/d5ddef04-d574-4c47-910a-71d56de33390)

**Comment by jlegind** [comment]
Christian Lange has approved the suggestion from email conversation 06-09-2023
[multi-specimen_warning.pdf](https://github.com/NHMDenmark/Mass-Digitizer/files/12595199/multi-specimen_warning.pdf)

**Comment by FedorSteeman** [comment]
> Christian Lange has approved the suggestion from email conversation 06-09-2023 [multi-specimen_warning.pdf](https://github.com/NHMDenmark/Mass-Digitizer/files/12595199/multi-specimen_warning.pdf)

This e-mail conversation is about the specimen condition and not the multi-specimen warning. 

Is the requirement to be according to https://github.com/NHMDenmark/Mass-Digitizer/issues/274#issuecomment-1711482993 ?

**Comment by FedorSteeman** [comment]
Anyways, estimated to take up to a day to implement, though probably a matter of hours.

**Comment by PipBrewer** [comment]
2 issues here: 1. The suggested text does not take into account both scenarios (mutispecimen and multiobject). 2. It has not been approved by those who requested it (the users and particularly Christian Lange). 

I will liaise with Chelsea and get back to you.

**Comment by PipBrewer** [comment]
Options:

**Comment by chelseagraham** [comment]
27 October

**Comment by chelseagraham** [comment]
@PipBrewer likes
![WARNING02](https://github.com/NHMDenmark/Mass-Digitizer/assets/108529653/c95fcf9a-7544-4aba-8e17-07559fb813f1)

I have sent an email to Christian Lange and Jens Soleberg. 
@Gomismis @nms419 @RebekkaML @AstridBVW @JesperMJensen please register your thoughts preferences here ASAP by 14 February

**Comment by Gomismis** [comment]
"This record will be linked to the previous record." If I had to choose, I would prefer the first option because it makes the most sense to me. However, the most important factors for me are:

1) That the warning don't pop up and block the app, because when digitizing, we aim to minimize interactions, clicks, and tabs in the app to optimize the digitization rate.

2) It should produce a sound. I believe this is important because 99% of the time, I forget to unclick the MOS button in the app it is caused by not looking up while digitizing (since all other settings remain the same).

**Comment by nms419** [comment]
"You are creating a linked record" or "Multi Radio Buttons are Activated" would make sense to me, but as long as the warning label is visually loud I don't think the exact wording matters much. 

If the selected multi radio button could be highlighted as well that would be great.

**Comment by AstridBVW** [comment]
I prefer the second option, "You are creating a linked record". I like the meaning of the first option but I think the text is too long.

I agree with Matilde, it is important that it does not block the flow. And I understand the idea behind adding a sound but I think most of us are digitising while using headphones and listening to either music or audiobooks. So I think the sound will have to be pretty loud for us to be able to hear it.

And yes, could be good to have the selected multi radio button highlighted.

**Comment by RebekkaML** [comment]
I also prefer "you are creating a linked record".

**Comment by JesperMJensen** [comment]
I agree with "You are creating a linked record" and also agree with Matildes two points on flow and sound.

**Comment by FedorSteeman** [comment]
@chelseagraham  As soon as there's agreement, please clarify or restate and assign this ticket to me.

**Comment by chelseagraham** [comment]
> @chelseagraham As soon as there's agreement, please clarify or restate and assign this ticket to me.

Will do, @FedorSteeman I am waiting for feedback from the collections managers. Probably after vinterferie :D

**Comment by FedorSteeman** [comment]
@chelseagraham Got feedback? I have reached this ticket and would like to start implementing soon.

**Comment by chelseagraham** [comment]
I nudged the collections managers yesterday for feedback. I'll let you know when I get it!

**Comment by chelseagraham** [comment]
Jens Soelberg thinks numbers 1-3 are fine, but agrees that number 2 is to the point and adequate for our users

**Comment by FedorSteeman** [comment]
So to sum up: Just a label corresponding to option nr 2 "WARNING! You are creating a linked record". This should become visible whenever an option other than single-specimen object is selected and made invisible when switching back again. I could not gauge a consensus or clear decision on a sound, so will leave that be for now.

**Comment by Gomismis** [comment]
> So to sum up: Just a label corresponding to option nr 2 "WARNING! You are creating a linked record". This should become visible whenever an option other than single-specimen object is selected and made invisible when switching back again. I could not gauge a consensus or clear decision on a sound, so will leave that be for now.

It would be really helpful for some of us if we could have the warning sound included. As I understand it that there's a concern about it not being heard by people with headphones, so I guess it won't bother those who don't want it.
